docs/docs/fundamentals/animations.md

```diff
@@ -96,6 +96,50 @@ This approach is more convenient in many cases, especially when view properties
 Also, keeping all the aspects of view styles and transitions colocated often makes it easier to keep control over your components' code.
 It forces you to have everything defined in one place vs scattered around the codebase allowing for animated transitions being triggered from anywhere.
 
+## Animations in inline styles
+
+For simple animations in `useAnimatedStyle` hook without any calculations like this:
+
+```js
+const animatedStyle = useAnimatedStyle(() => {
+  return {
+    width: withSpring(width.value),
+  };
+});
+<Animated.View style={animatedStyle} />
```

Comment:
```suggestion
function SomeComponent() {
  const animatedStyle = useAnimatedStyle(() => {
    return {
      width: withSpring(width.value),
    };
  });

  return <Animated.View style={animatedStyle} />;
}
```

Reply:
After reading through the docs the `useAnimatedStyle` is introduced and explained in `shared-values.md` section. So I'd need to rewrite a large part of `shared-values.md` and `animations.md` pages. But we're going to rewrite all the docs soon anyway with different sections, right? Is it worth it rewriting it now if it's going to be rewritten soon anyway?